veba-bom.json

```diff
@@ -5,7 +5,7 @@
     "vmware-event-router": {
         "gitRepoTag": "v0.5.0",
         "containers": [{
-            "name": "vmware/veba-event-router",
+            "name": "projects.registry.vmware.com/veba/veba-event-router",
```

Comment:
The current VEBA release process pushes to Docker only. This would be an issue unless we manually push to the new registry or change the release process to also push to this one. WDYT?

Reply:
Correct. Right now, its just a manual push to centralize all VMware-based container images. Perhaps we should have a new Issue raised to port the logic to push from source code to VMware Harbor rather than Docker?

Reply:
Good point!